src/components/Admin/SelectedUserModal.js

```diff
@@ -114,11 +124,9 @@ const SelectedUserModal = ({showModal, closeModal, removeEntry=false, selectedAl
                 {selectedAlterUser.role && selectedAlterUser.role.isAffiliate && 
                     <FormField name="affiliateCode" placeholder="Affiliate Code" type="text" />
                 }
-
-                {!removeEntry && <FormField name="referralCode" placeholder="Referral Code" type="text" />}
```

Comment:
Why was this deleted?

And when put back, it also shows when the account isUser

Reply:
I thought about it again, `referralCode` should only be set when registering, there's no need to be able to set that field after the user has registers.

Reply:
The admin wants to be able to edit it. So on edit users, it's a field that the admin can alter.